geometry/Overview.bs

```diff
@@ -2117,15 +2132,26 @@ href="https://www.w3.org/TR/2014/WD-geometry-1-20140918/">18 September 2014 Work
 
    <li><p>Added no-argument constructor.
 
-   <li><p>Defined <code>window.WebKitCSSMatrix</code> to alias {{DOMMatrix}}.
+   <li><p>Defined {{WebKitCSSMatrix}} to be a legacy window alias for {{DOMMatrix}}.
   </ul>
 
  <li><p>Exposed {{DOMPointReadOnly}}, {{DOMPoint}}, {{DOMRectReadOnly}}, {{DOMRect}}, {{DOMQuad}},
- {{DOMMatrixReadOnly}} and {{DOMMatrix}} to {{Window}} and {{Worker}}. Defined cloning of the
- interfaces.
+ {{DOMMatrixReadOnly}} and {{DOMMatrix}} to {{Window}} and {{Worker}}. Defined structured
+ serialization of the interfaces. In workers, {{DOMMatrix}} and {{DOMMatrixReadOnly}} do not support
```

Comment:
Structured serialization seems separate from this mostly-about-workers bullet point